tensorboard/plugins/example/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Generated by running `python setup.py develop`.
```

Comment:
```
Requires license information in below file(s):
tensorboard/plugins/example/.gitignore
```

😅

Reply:
You know, I _almost_ ran that test locally! :-) Thanks.